### ISSUE TITLE
Is possible to mute the duplicate `load` log in the terminal? 

### ISSUE THREAD

**Issue by callmeberzerker** [joshnuss/vite-plugin-notifier#2]
# Description

Currently whenever a file is changed in the `terminal` the following lines are displayed:

```
2:32:37 PM [vite] hmr update /src/shared/form/control/DateTimePickerField.tsx
load /Users/my-user/projects/the-project/src/shared/form/control/DateTimePickerField.tsx
```

The first one is from `vitejs` itself and the second it is from the plugin I believe.

**Comment by joshnuss** [comment]
Hi @callmeberzerker,

I'm not seeing that on my end, are you on the latest version? I updated a few times over the past few hours.

This is what it outputs on an example project:

```
svelte-test > npm run dev

> svelte-test@0.0.0 dev
> vite

Port 3000 is in use, trying another one...

  vite v2.4.4 dev server running at:

  > Local: http://localhost:3001/
  > Network: use `--host` to expose

  ready in 283ms.

8:45:57 a.m. [vite] hmr update /src/App.svelte
8:46:11 a.m. [vite] hmr update /src/App.svelte (x2)
```

**Comment by callmeberzerker** [comment]
Hi @joshnuss,

I am using `vite@2.4.4` and the latest version of the plugin `0.1.4`. Not sure what's the cause here... 🕵️ 

In my case on `dev` it lists all the files with the `load` prefix, and then when I change a file -> it prints both the [vite HMR] and load message for that file as described in the first message.

If I remove the plugin there is no `load` prefix nor duplicated statements.

**Comment by callmeberzerker** [comment]
Just tested it on my Windows machine (previously I was on a Mac) with a separate new project. It's still happening... I will publish repro repo soon. It's weird because I don't see any `load ...` statement sin the plugin source code. 😕 

Here is a sample vite `react-ts` template project with only this plugin: https://github.com/callmeberzerker/example-notifier

When you start the project you will see something similar to:

```
load E:/Projects/example-notifier/node_modules/.pnpm/vite@2.4.4/node_modules/vite/dist/client/client.mjs
/vite-plugin-notifier
/vite-plugin-notifier
/vite-plugin-notifier
load /vite-plugin-notifier
/vite-plugin-notifier
load E:/Projects/example-notifier/src/main.tsx
load E:/Projects/example-notifier/node_modules/.pnpm/vite@2.4.4/node_modules/vite/dist/client/env.mjs
load E:/Projects/example-notifier/src/index.css
load E:/Projects/example-notifier/src/App.tsx
load E:/Projects/example-notifier/src/App.css
/vite-plugin-notifier

```

**Comment by joshnuss** [comment]
Thanks for repro, I found the problem.

I had some uncommitted `console.log`'s when i did `npm publish`, forgot that it uses the current working dir (doh).

I've cut a new version `0.1.5` and it's now fixed in 1a97dccf7f98998b8b7974c945edb830b9803c1e 

Thanks for reporting!

**Comment by callmeberzerker** [comment]
Awesome thanks! :)
